tests/utils/git_test.py

```diff
@@ -17,88 +17,76 @@
 
 import logging
 import os
-import shutil
 from unittest import mock
 
 import pytest
+from git import Repo
 
 from esrally import exceptions
-from esrally.utils import git, process
+from esrally.utils import git
+
+
+def commit(repo, *, date=None):
+    file_name = os.path.join(repo.working_dir, "test")
+    with open(file_name, "wb"):
```

Comment:
Maybe just me, but I was a little confused at first.  
```suggestion
    file_name = os.path.join(repo.working_dir, "test")
    # creates an empty file
    with open(file_name, "wb"):
```